vertical-pod-autoscaler/README.md

```diff
@@ -387,6 +388,11 @@ vpa-post-processor.kubernetes.io/{containerName}_integerCPU=true
 
 These options cannot be used together and are mutually exclusive. 
 
+ ### Setting the webhook failurePolicy
+
+ It is possible to set the failurePolicy of the webhook to `Fail` by passing `--webhook-failure-policy-fail=true` to the VPA admission controller.
+ Please use this option with caution as it may be possible to break Pod creation if there is a failure with the VPA.
+Using it in conjunction with `--ignored-vpa-object-namespaces` or `--vpa-object-namespace` to reduce risk.
```

Comment:
Can we be explicit here that if not using `--vpa-object-namespace` to isolate this to a set of namespaces that we recommend minimally setting `--ignored-vpa-object-namespaces=kube-system`?

Reply:
Good idea! Added in 205b978b72